vagrant/ansible/roles/node.setup/tasks/main.yml

```diff
@@ -28,3 +28,20 @@
       - lvm2
       - firewalld
       - libselinux-python
+
+- name: Install yum copr plugin
+  yum:
+    name: yum-plugin-copr
+    state: latest
+
+- name: add copr to get compat-gnutls34 (needed for centos 7)
```

Comment:
These CentOS 7 specific changes should go into a block with a when condition checking for CentOS 7. This will make it easier when we start testing for CentOS 8 as well.

I found the following variable you can test for.
https://www.middlewareinventory.com/blog/ansible-facts-list-how-to-use-ansible-facts/
    when:
      - ansible_facts['distribution'] == "CentOS"
      - ansible_facts['distribution_major_version'] == "7"

Reply:
oops. Done already.